manifests-servicemesh/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
       The Service Mesh Operator provides means for installing, configuring, and managing an instance of Red Hat OpenShift Service Mesh.
```

Comment:
Change this to read:

The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh.

Reply:
I am OK with the other lines you asked me to check @dmitri-d.